scene/main/timer.cpp

```diff
@@ -105,6 +117,16 @@ bool Timer::has_autostart() const {
 	return autostart;
 }
 
+void Timer::set_max_timeouts_per_tick(int p_max_timeouts) {
+	ERR_FAIL_COND_MSG(p_max_timeouts <= 0, "Maximum timeouts per tick must be greater than or equal to 1.");
```

Comment:
```suggestion
	ERR_FAIL_COND_MSG(p_max_timeouts <= 0, "Maximum timeouts per tick must be greater than zero.");
```
Matching the other one